python/ray/__init__.py

```diff
@@ -4,11 +4,7 @@
 import os
 import sys
 
-# For cases like docs builds, we want the default logging config.
-skip_reset = os.environ.get("SKIP_LOG_RESET", False)
-if not skip_reset:
-    log.generate_logging_config()
-
+log.generate_logging_config()
```

Comment:
Thanks for all the detailed investigation!

I have a question with this change: wo we will always generate logging config - which we were not? Is this change causing the build to fail?

Reply:
Yes, we will always generate logging config, which is what we want. The original problem we were addressing here is that the sphinx build loggers get disabled if you don't have `'disable_existing_loggers': False` in the call to `dictConfig` to set up the loggers. I don't think that's the issue here.

RTD's fail-on-warning was only enabled last week; I think that's relevant here.